astropy/io/fits/diff.py

```diff
@@ -1397,19 +1397,19 @@ def _report(self):
             for name in self.diff_column_names[0]:
                 format = self.diff_columns[0][name.lower()].format
                 self._writeln(' Extra column {} of format {} in a'.format(
-                                name, format))
```

Comment:
why not do f-strings here, too?

Reply:
pyupgrade does not detect those, and there are many many more.